bin/make-zip.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Build a installable plugin zip
+
+output 2 "Creating archive... 🎁"
+
+build_files=$(ls dist/*/*.{js,css})
+ZIP_FILE=$1
+
+zip -r ${ZIP_FILE} \
```

Comment:
It might be nice to zip everything while excluding folders and files that should not be included instead of including all the files needed.

I imagine this may be a source of hiccups in testing if a new root file or vendor is added.

Reply:
@joshuatf Added in `886e209`.